lib/SecondPage.dart

```diff
@@ -0,0 +1,223 @@
+import 'package:flutter/material.dart';
+import 'package:flutterproject/ThirdPage.dart';
+import 'package:flutterproject/databasehelper.dart';
+
+class SecondPage extends StatefulWidget {
```

Comment:
pageler için lib klasörünün içerisine view klasörü oluşturabilirsin

Reply:
Yapıldı